src/components/LogoutButton/LogoutButton.css

```diff
@@ -0,0 +1,3 @@
+/* .LogoutButton {
```

Comment:
Was this commented out as in you opted to go a different route or as in it need to be added back later?

Reply:
Ah yes, decided to do something different and just didn't delete. I'll clean that up before we merge.